browser/src/robocorp/browser/_config.py

```diff
@@ -32,45 +32,31 @@ def __init__(
     ):
         """
         Args:
-            browser_engine:
-                Browser engine which should be used
+            browser_engine: Browser engine which should be used
                 default="chromium"
                 choices=["chromium", "chrome", "chrome-beta", "msedge",
                          "msedge-beta", "msedge-dev", "firefox", "webkit"]
```

Comment:
I see this didn't got updated as the other same parameter you refactored in `__init__.py`. Shouldn't these `Args:` be aligned with the content there?